ui/app/models/pki/pki-key-engine.js

```diff
@@ -0,0 +1,10 @@
+import Model, { attr } from '@ember-data/model';
+
+export default class PkiKeyEngineModel extends Model {
+  @attr('string', { readOnly: true }) backend;
```

Comment:
I may be reading the wrong docs - but should there also be a `type` and `key_bits` attribute? [API Docs](https://www.vaultproject.io/api-docs/secret/pki#parameters-10)

(Disregard if this model is still WIP!)

Reply:
You're not wrong, but yep, model is still a WIP. I only added the read and list params here. Something I've been doing for these list views. I should have left a comment about it.